Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -166,6 +166,13 @@
                 
                 If you don't see your user, delete the app from the simulator, and in the menu go to Device > Erase All Content and Settings (which resets your simulator), and try to run again
             }
+            
+            @Step {
+                If you receive the following error when you launch the emulator:  _'firebase-tools no longer supports Java version before 11. Please upgrade to Java version 11 or above to continue using the emulators'_
+
+                The openJDK install failed and you will have to install the latest JDK manually. You can download the latest version here [JDK23](https://www.oracle.com/java/technologies/downloads/#jdk23-mac) 
+            }
+            
```

Comment:
I moved this, because your step is really after they've run the emulator, not before so I moved it in the logic flow of following the tutorial.

And note in DocC you only have two sections in a `@Step` just the direction and then optional note, you originally had multiple lines, see screenshot, which don't render, and create warnings in Xcode

![Screenshot 2024-10-06 at 10 30 16 AM](https://github.com/user-attachments/assets/b0a70e6b-26f6-447e-a0ed-4fa38af99c61)

![Screenshot 2024-10-06 at 10 30 20 AM](https://github.com/user-attachments/assets/569f1a6e-1b64-469c-bd39-37a3fa335b0e)